fineract-provider/config/swagger/fineract-input.yaml

```
@@ -22,7 +22,7 @@ info:
     name: Apache 2.0
     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
 servers:
-- url: /fineract-provider/api/v1
+- url: https://demo.openmf.org/fineract-provider/api/v1
```

This currently appears to be down? But even if it was up, I would much prefer this to use `fineract.dev` instead:

```suggestion
- url: https://demo.openmf.org/fineract-provider/api/v1
```

@Grandolf49 see https://www.fineract.dev for background... OK for you?

---

@Grandolf49 in the interest of moving forward with this, I'll add this to your branch by self-accepting this suggestion to your PR, and if the build still passes, I'll merge this. Please comment here if you disagree with changing this URL, and we can discuss it further. Whatever server we use server is IMHO "wrong" anyway - we should use the LOCAL server - otherwise this will never work for new APIs being added... I'll comment about this on  FINERACT-838.

---

Yeah sure! This works for me!